src/generic_ui/polymer/link.ts

```diff
@@ -0,0 +1,6 @@
+Polymer({
+  handleKey: function() {
+    this.click();
```

Comment:
My assumption is it handles if the event was set up with `on-click`.  I took the behaviour for this from the way it is handled in `paper-button`.